packages/@aws-cdk/cloud-assembly-schema/schema/cloud-assembly.version.json

```diff
@@ -1 +1 @@
-{"version":"7.0.0"}
\ No newline at end of file
+{"version":"9.0.0"}
```

Comment:
No need to bump this again for different revisions of the same PR. I know the tool will tell you to, but ignore it/change it back.